flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/SqlClientTest.java

```diff
@@ -176,12 +176,14 @@ public void testVerboseErrorMessage() throws Exception {
     public void testInitFile() throws Exception {
         List<String> statements =
                 Arrays.asList(
-                        "CREATE TABLE source ("
+                        "-- Define Table \n"
+                                + "CREATE TABLE source ("
                                 + "id INT,"
                                 + "val STRING"
                                 + ") WITH ("
                                 + "  'connector' = 'values'"
-                                + ");\n",
+                                + "); "
+                                + "-- Define Table \n",
                         "SET key = value;\n");
```

Comment:
Could you also add some comments after the `SET`? Because it fails to parse the statement when comment is around SQL Client commands.

Reply:
just like this "SET key = value; -- define set \n" right ?

Reply:
Yes.